src/test/CMakeLists.txt

```diff
@@ -254,6 +254,7 @@ add_executable(ceph_test_librgw_file
   )
 set_target_properties(ceph_test_librgw_file PROPERTIES COMPILE_FLAGS
   ${UNITTEST_CXX_FLAGS})
+target_include_directories(ceph_test_librgw_file PUBLIC ${FCGI_INCLUDE_DIR})
```

Comment:
nit, `PRIVATE` would suffice here. as no need to populate ceph_test_librgw_file's INCLUDE_DIRECTORIES to other targets.